pilot/pkg/bootstrap/server.go

```diff
@@ -1249,5 +1269,6 @@ func (s *Server) initWorkloadTrustBundle(args *PilotArgs) error {
 // It return true only if istiod certs is signed by Kubernetes and
 // workload certs are signed by external CA
 func (s *Server) isDisableCa() bool {
-	return features.PilotCertProvider == constants.CertProviderKubernetes && s.RA != nil
+	return features.PilotCertProvider == constants.CertProviderKubernetes ||
```

Comment:
can you put parenthesis or split it up? its confusing to have `a || b && c` since you need to remember precedence rules

Reply:
Sure, fixed